### ISSUE TITLE
Repeater module

### ISSUE THREAD

**Issue by Florrpan** [beaverbuilder/feature-requests#118]
### The Repeater module
A module where you can visually construct the repeater field and style the outcome.
You could construct custom restaurant menus, item-lists, how-to, teams and much more. 

The module could either be set to repeat items inside itself or fetch information from custom fields, making it useful for many different types of layouts.

I've always wanted a module or generally better modules that utilizes custom fields repeater functions. There's not much to go on right now except for coding the repeater yourself with BB shortcodes. But this takes the function away from users who isn't used to HTML/CSS. Your average users who uses the builder the simple way.

Basically, the module could act as an one-for-all, a powerful module where creativity is the only limitation.

BB core modules are lacking a lot of creative modules that comes with 3rd party developers. This could fill that gap and satisfy a lot of different users.

**Comment by grafixone** [comment]
Repeater module PLEASE

**Comment by jaycanuck74** [comment]
Yes please! I'm currently in the middle of a project that greatly requires some sort of Module-ACF Repeater support

**Comment by zackpyle** [comment]
Why hasn't something like this been added to Beaver already? https://developers.elementor.com/how-to-add-a-repeater-control-to-elementor-add-on/

It should be baked into core...

**Comment by Florrpan** [comment]
Any news on this? Have we considered it?

**Comment by codente** [comment]
This is in Progress!  Currently expected for Themer 1.6 with Themer 1.5 to come as a pre-release very soon.

**Comment by codente** [comment]
The Loop module is now out in Themer 1.5.  It requires BB 2.9 which is also out now :)
